src/main/kotlin/com/jakewharton/gradle/dependencies/treeDiff.kt

```diff
@@ -18,7 +18,7 @@ fun dependencyTreeDiff(old: String, new: String): String {
 }
 
 private fun findDependencyPaths(text: String): Set<List<String>> {
-	val dependencyLines = text.split('\n')
+	val dependencyLines = text.split(Regex("\r?\n"))
```

Comment:
Probably better to use https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/lines.html

Reply:
Oh of course :) will fix

Reply:
rebased on trunk and changed in https://github.com/JakeWharton/dependency-tree-diff/pull/11/commits/eb7794a1fc71917cf7df4aae9ad673bb47232028